docs/usage/templates.md

```diff
@@ -110,6 +110,12 @@ Returns `true` if at least one expression is `true`.
 
 `{{#if (or isPatch isSingleVersion}}Small update, safer to merge and release.{{else}}Check out the changelog for all versions before merging!{{/if}}`
 
+### includes
+
+Returns `true` if the value is included on the list given.
+
+`{{#if (includes depTypes "dependencies")}}production{{else}}notProduction{{/if}}`
```

Comment:
Does this work yet? If not then we need a different example such as labels

Reply:
Good catch! I would say it does not work yet based on my findings on the PR https://github.com/renovatebot/renovate/pull/27939. So, I will update the tests and example and test it!

Reply:
Done!